MediaBrowser.XbmcMetadata/Savers/BaseNfoSaver.cs

```diff
@@ -353,7 +353,7 @@ public static void AddMediaInfo<T>(T item, XmlWriter writer)
 
                 if (!string.IsNullOrEmpty(stream.Language))
                 {
-                    // https://emby.media/community/index.php?/topic/49071-nfo-not-generated-on-actualize-or-rescan-or-identify
+                    // https://jellyfin.media/community/index.php?/topic/49071-nfo-not-generated-on-actualize-or-rescan-or-identify
```

Comment:
Same as above

Reply:
Good catch, I'll sort this and above.

Reply:
Sorted with: https://github.com/jellyfin/jellyfin/pull/332/commits/2afacbc04099b453ac235b7564f9f7066da9d4ab